cmd/monaco/deploy/command.go

```diff
@@ -39,6 +41,7 @@ func GetDeployCommand(fs afero.Fs) (deployCmd *cobra.Command) {
 		ValidArgsFunction: completion.DeployCompletion,
 		PreRun:            cmdutils.SilenceUsageCommand(),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
```

Comment:
I would put this in line 53, that is just read it there

Reply:
done